docs/02_Introduction_into_Metafacture-Flux.md

```diff
@@ -268,16 +266,16 @@ Now take some time and play around a little bit more and use some other modules.
 What you see with the modules `encode-formeta` and `write` is that modules can have further specification in brackets.
 These can eiter be a string in `"..."` or attributes that define options as with `style=`.
 
-One last thing you should learn on an abstract level is to grasp the general idea of Metafacture Flux workflows is that they have many different moduls through which the data is flowing.
-The most abstract and most common process resemble the following steps:
+One last thing you should learn on an abstract level to grasp the general idea of Metafacture Flux workflows is that they have many different modules through which the data is flowing.
+The most abstract and most common process is resembled by the following steps:
```

Comment:
```suggestion
The most abstract and most common process resemble the following steps:
```

Reply:
Thx for the suggestion. I just added the 3.person singular `s` at the verb.